tests/devapps/MauiApps/MauiAppB2C/Features/LogOn/UserContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace UserDetailsClient.Core.Features.LogOn
+{
+    public class UserContext
```

Comment:
If this is a dev app, why do you complicate it so much? The MSAL team understand AuthenticationResult. I don't see the point of another class.